test/performance/README.md

```diff
@@ -21,3 +23,30 @@ Their name reference the test where they come from, currently `get-*.csv` and `c
 
 -   `.txt` files contain summaries of Locust, the time unit is milliseconds.
 -   `.csv` files contain values from Prometheus, the CPU unit is in milliCPU, the memory unit is bytes.
+
+## Merging results from GitHub Workflows
+
+To merge test results from GitHub Workflows you need to run the following command:
+
+```bash
+export GH_TOKEN="a personal github token"  # Needs read permission for workflows so it can download artifacts
+bash test/performance/merge.sh
+```
+
+A bunch of output will get printed into stdout, but to visualize the results run:
+
+```
+# You need to install `gnuplot` first
```

Comment:
Could you move this comment outside the code section. Maybe we can add a prerequisite list, what do you think?

Reply:
Done